test/libsolidity/util/TestFunctionCall.h

```diff
@@ -116,6 +126,13 @@ class TestFunctionCall
 		std::string const& _linePrefix = ""
 	) const;
 
+	/// Formats gas usage expectations one per line
+	std::string formatGasExpectations(
+		std::string const& _linePrefix,
+		bool const _renderGasCostResult,
+		bool const _debugGasPrint
+	) const;
```

Comment:
How about these names?
```suggestion
	std::string formatGasExpectations(
		std::string const& _linePrefix,
		bool _useActualCost,
		bool _showDifference
	) const;
```

But it would be much more readable as an enum: `ExpectedCost`, `ActualCost`, `ActualCostAndDiffference`.

Also, I'm not sure if there's much sense in declaring these `bool`s as `const`. I'd personally prefer if they were `const` by default but since they're not, adding `const` manually everywhere seems a bit tedious.